src/Expectation/UrlsHelper.php

```diff
@@ -0,0 +1,125 @@
+<?php
```

Comment:
All `@param` declarations in this file are missing the `type` information, which makes them kind of useless.

Reply:
All `@param` are mixed anyway. The type is checked inside the function via `is_string` and similar. So you can pass whatever. Mixed is implicit so to me that's fine. 
If we would add Psalm or similar, as soon I would declare `@param string` and then do `is_string()` checks inside the function Psalm would rightfully complain for "redundant check". 
I prefer to keep the check at runtime so all the parameters are mixed, and IMO there's no point in typing "mixed" all over the place.
Still, I like to keep the doc block for "visual help". I personally find it hard to digest a wall of code without some "calming" doc block in between functions.